IMG_webp.c

```diff
@@ -62,37 +63,47 @@ static struct {
 } lib;
 
 #ifdef LOAD_WEBP_DYNAMIC
```

Comment:
this should be changed to `#if defined(LOAD_WEBP_DYNAMIC) && defined(LOAD_WEBPDEMUX_DYNAMIC)`

Reply:
force pushed these changes